src/watcher.js

```diff
@@ -99,10 +99,13 @@ class Watcher extends SafeEventEmitter {
                         this.emit('load.chat');
                         break;
                     case 'directory.following.index':
+                        this.emit('load.following');
```

Comment:
The reason for adding the new event is because if you switch from another tab back to the index tab the event wont fire:
```
// Switching between tabs in following page
if (lastRoute.substr(0, 19) === 'directory.following') break;
```
It makes the event unreliable for this modules and I didn't want to mess with the existing event and possibly break a module that relies on it.